src/layouts/sidebar-sidecar/utils/prepare-nav-data-for-client.ts

```diff
@@ -162,7 +162,7 @@ async function prepareNavNodeForClient({
 		 * we could add a warning or error here, or resolve this
 		 * through content conformance efforts.
 		 */
-		const pathWithIndexFix = node.path == 'index' ? '' : node.path
+		const pathWithIndexFix = node.path.endsWith('/index') ? '' : node.path
```

Comment:
Is there a spike task for investigation a better solution? I would suggest linking to that task in a code commit with a #FIXME.

Reply:
No, there is not currently a task. I will create one and flag it here.

Reply:
fixed in https://github.com/hashicorp/dev-portal/pull/2542/commits/538d888d2b00ab17b6a17dd6d4e3c7543fd16c5e